src/transformers/models/bart/modeling_bart.py

```diff
@@ -1125,6 +1125,11 @@ def __init__(self, config: BartConfig):
         # Initialize weights and apply final processing
         self.post_init()
 
+    def _tie_weights(self):
```

Comment:
Ah nice, noticed this as well actually :sweat_smile: 
https://github.com/huggingface/transformers/pull/27203/files#r1378948159